src/renderer/components/IsUnlocked.js

```diff
@@ -91,6 +93,15 @@ const IsUnlocked = ({ children }: Props) => {
       try {
         if (!isAccountDecrypted) {
           await setEncryptionKey("app", "accounts", inputValue.password);
+          try {
+            // The password worked for db, it should be the same for libcore
+            await setLibcorePassword(inputValue.password);
+          } catch (e) {
```

Comment:
i'm not sure about the idea to generically catch all error cases. we should specifically expect some error cases no?